loader/loader.go

```diff
@@ -388,8 +414,6 @@ func (l *Loader) toGunkPackage(lpkg *packages.Package) *GunkPackage {
 			}
 		}
 	}
-
-	return pkg
```

Comment:
Is this now an implicit return in Go if it gets to the end of the function and their is no return?

Reply:
The old func was `func(*packages.Package) *GunkPackage`, and it's now `func(*GunkPackage)`. So it just modifies the package in-place. The returns now simply skip extra work, and don't actually return anything.

Reply:
Ahh, not sure how I didn't see that. Sorry.

Reply:
It's confusing because the context shows:

```
@@ -388,8 +414,6 @@ func (l *Loader) toGunkPackage(lpkg *packages.Package) *GunkPackage {
```

...which isn't useful at all, because that is the *removed* line.